src/test/java/calculator/domain/CalculatorTest.java

```diff
@@ -0,0 +1,44 @@
+package calculator.domain;
+
+import org.assertj.core.api.ThrowableAssert.ThrowingCallable;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class CalculatorTest {
+
+    @Test
+    void create_calculator() {
+        Numbers numbers = new Numbers(List.of(1, 2, 3));
+        Operators operators = new Operators(List.of(Operator.ADD, Operator.SUBTRACT));
```

Comment:
import static을 통해 가독성을 높여보시면 어떨까요?
```suggestion
        Operators operators = new Operators(List.of(ADD, SUBTRACT));
```